File: PYTHON/student_grade.py
student_count = int(input("Enter number of students: "))
subject_count = int(input("Enter number of subjects: "))

top_student = ""
top_score = -1

student_index = 1
while student_index <= student_count:
    print(f"\nStudent {student_index}")
    total = 0
    subject_index = 1

    while subject_index <= subject_count:
        score = float(input(f"Enter score for Subject {subject_index}: "))
        total += score
        subject_index += 1

    average = total / subject_count
    print(f"Total: {total} Average: {average:.2f}")

    if total > top_score:
        top_score = total
        top_student = f"Student {student_index}"

    student_index += 1

print(f"\nTop Performer: {top_student} with total score of {top_score}")

File: JAVA/StudentGrade.java
import java.util.Scanner;

public class StudentGrade {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

System.out.print("Enter number of students: ");
int studentCount = input.nextInt();

System.out.print("Enter number of subjects: ");
int subjectCount = input.nextInt();

String topStudent = "";
double topScore = -1;

int studentIndex = 1;
while (studentIndex <= studentCount) {
System.out.println("\nStudent " + studentIndex);
double total = 0;

int subjectIndex = 1;
while (subjectIndex <= subjectCount) {
System.out.print("Enter score for Subject " + subjectIndex + ": ");
double score = input.nextDouble();
total += score;
subjectIndex++;
}

double average = total / subjectCount;
System.out.println("Total: " + total + " Average: " + String.format("%.2f", average));

if (total > topScore) {
topScore = total;
topStudent = "Student " + studentIndex;
}

studentIndex++;
}

        System.out.println("\nTop Performer: " + topStudent + " with total score of " + topScore);
        input.close();
    }
}
